docs/en_US/install/wiiu.md

```diff
@@ -55,6 +55,11 @@ Bleeding edge builds have not been widely tested, and do not guarantee any stabi
 
 <img src="/assets/images/docs/install/wiiu/aroma/inkay-nightly-link.png" width="100%">
 
+## Nimble
+Nimble is a Aroma plugin that redirects the PolicyList servers to Pretendo's servers. This makes certain games (such as Splatoon) work with online play. Download the latest version from [Nimble's latest release](https://github.com/PretendoNetwork/Nimble/releases/latest) on GitHub and download the `30_nimble.rpx` file from the GitHub page.
```

Comment:
My previous comment on this may have been confusing. Games like Splatoon download their content through BOSS (SpotPass). Inkay already contains patches for BOSS, so those games should already be covered.

Nimble specifically patches the policy list. This is not on it's own server, it is also part of the BOSS server. But the reason it's in its own patch is because the console downloads the policy list too early in the boot process for Inkay to handle it. This is then used by tasks like Wara Wara Plaza to download content at system boot.

This is why I tagged @ashquarky, @DaniElectra and @shutterbug2000 for opinions before with regards to merging the patches and to confirm if the Nintendo shut down affected things on our end in unexpected ways. Please do not mark this as resolved until that has been discussed

Reply:
Alright, let me know on any updates you get.

Reply:
I'm not aware of any shutdown-related breakage other than the policylist being all suspended now, which Nimble fixes. I was under the impression some games (like Splatoon) actually use the policylist though? Either way we need Nimble now.

Nimble needs to be a separate file for boring technical reasons (Aroma plugins run too *late* in the boot process) but we could certainly offer a zip with everything already arranged in it, like aroma.foryour.cafe does.

Reply:
> I'm not aware of any shutdown-related breakage other than the policylist being all suspended now, which Nimble fixes. I was under the impression some games (like Splatoon) actually use the policylist though? Either way we need Nimble now.
> 
> 
> 
> Nimble needs to be a separate file for boring technical reasons (Aroma plugins run too *late* in the boot process) but we could certainly offer a zip with everything already arranged in it, like aroma.foryour.cafe does.

I know it needs to be separate, but any chance they could be built at the same time from the same repo? Or would trying to merge them be too much. It's been a while since I've looked at the Nimble repo

Reply:
They could be folders in the same repo I suppose, there's not really any overlap code-wise since Inkay is WUPS and Nimble is basically a normal homebrew app

I assume you want this so that we have one GitHub release?

Reply:
> They could be folders in the same repo I suppose, there's not really any overlap code-wise since Inkay is WUPS and Nimble is basically a normal homebrew app
> 
> 
> 
> I assume you want this so that we have one GitHub release?

Yeah that would be the goal. But if it's not feasible that's 100% fine. An installer would solve these issues too